test/core/crypto/hasher/hasher_test.cpp

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <stdio.h>
+
+#include "common/blob.hpp"
+#include "crypto/hasher/hasher_impl.hpp"
+
+/**
+ * Hasher fixture
+ */
+class Hasher : public testing::Test {
+ public:
+  using Buffer = kagome::common::Buffer;
+
+  ~Hasher() override = default;
+
+  Hasher() {
+    hasher_ = std::make_shared<kagome::hash::HasherImpl>();
+  }
+
+  auto hasher() {
+    return hasher_;
+  }
+
+  /**
+   * useful function for convenience
+   */
+  template <int size>
+  static Buffer blob2buffer(const kagome::common::Blob<size> &blob) noexcept {
+    Buffer out;
+    out.put({blob.data(), static_cast<long>(blob.size())});
+    return out;
+  }
+
+  static Buffer string2buffer(const std::string_view &view) noexcept {
+    Buffer out;
+    out.put(view);
+    return out;
+  }
+
+ private:
+  std::shared_ptr<kagome::hash::Hasher> hasher_;
```

Comment:
just make this field protected and remove `hasher` method. In test fixtures there is no need to write getters for private fields

Reply:
made protected